lib/gui/pages/main/controllers/main.js

```diff
@@ -19,6 +19,7 @@
 module.exports = function(
   SelectionStateModel,
   DrivesModel,
+  DriveInfoModalService,
```

Comment:
The main page should declare the `drive-info-modal` component as a dependency before attempting to use it.